python/ql/test/experimental/query-tests/Security/CWE-295/urlopen.py

```diff
@@ -0,0 +1,26 @@
+import urllib.request
+
+#Simple cases
+urllib.request.urlopen('https://semmle.com', "/path/to/cert/") # BAD (does not assign cert path to "cafile")
+urllib.request.urlopen('https://semmle.com', cafile="/path/to/cert/")
+urllib.request.urlopen('https://semmle.com') # BAD
+urllib.request.urlopen('https://semmle.com', cafile=None) # BAD
+urllib.request.urlopen('http://semmle.com', cafile=None) # OK (not https link)
+
+# Simple flow
+urlopen = urllib.request.urlopen
+urlopen('https://semmle.com', cafile="/path/to/cert/") # GOOD
+urlopen('https://semmle.com', cafile=None) # BAD
```

Comment:
It would be nice to see other cases with local flow for both the URL and the `None` (which I believe may have been present in the other PR).

(Also, this is a minor nit, but on line 12, `/path/to/cert/` does not look like a path to a `cafile`, but rather a `capath`.)